packages/core/src/store/pin-store.ts

```diff
@@ -5,18 +5,29 @@ import { RunningState } from '../state-management/running-state.js'
 import { Model } from '@ceramicnetwork/stream-model'
 import { IKVStore } from './ikv-store.js'
 import { StreamStateStore } from './stream-state-store.js'
+import { Semaphore } from 'await-semaphore'
+
+const DEFAULT_CONCURRENT_PIN_LIMIT = 10
```

Comment:
not really sure what's a reasonable default here.  @v-stickykeys do you have any intuition here?